migrations/20240210165254_variants__reference_course_id.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE variants
+ADD COLUMN IF NOT EXISTS course_id bigint;
+
+ALTER TABLE variants
+ADD CONSTRAINT variants_course_id_fkey FOREIGN KEY (course_id) REFERENCES pl_courses (id) ON UPDATE CASCADE ON DELETE CASCADE;
```

Comment:
Not that it really matters, but you can add this directly in the ADD COLUMN query (and also add NOT NULL).
```suggestion
ALTER TABLE variants
ADD COLUMN IF NOT EXISTS course_id bigint NOT NULL REFERENCES pl_courses (id) ON UPDATE CASCADE ON DELETE CASCADE;
```

Reply:
Oh, that's cool! When you do it that way, does it automatically generate the constraint name for you?